Makefile

```diff
@@ -3,57 +3,63 @@ HOSTNAME=registry.terraform.io
 NAMESPACE=leanspace
 NAME=leanspace
 BINARY=terraform-provider-${NAME}
-VERSION?=0.4.0
+VERSION?=10.0.0
```

Comment:
We can choose any version here or it was just for testing?

Reply:
you can choose any version
Here it means by default if you don't provide the VERSION environment variable it will be equal to this value otherwise it used the environment variable's value